pom.xml

```diff
@@ -27,7 +27,7 @@
         <environment-reader-library.version>1.3.2</environment-reader-library.version>
         <!-- Logging -->
         <log4j-bom.version>2.17.0</log4j-bom.version>
-        <structured-logging.version>1.9.12</structured-logging.version>
+        <structured-logging.version>1.9.14</structured-logging.version>
```

Comment:
should we go back to 1.9.12 for now?

Reply:
I've merged `master` branch in and picked its version, i.e. 1.9.12

Reply:
👍